sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -856,20 +864,31 @@ private[hive] class HiveClientImpl(
   }
 
   def reset(): Unit = withHiveState {
```

Comment:
Thank you for pinging me, @wangyum . 
Shall we make another PR for this `reset` change?

Reply:
OK.

Reply:
I believe this `reset` PR can be merged before this `runHive` PR.